native/bls_nif/src/lib.rs

```diff
@@ -28,4 +28,30 @@ fn sign<'env>(
     Ok(bytes_to_binary(env, &bytes))
 }
 
-rustler::init!("Elixir.Bls", [sign]);
+#[rustler::nif]
+fn fast_aggregate_verify<'env>(
+    public_keys: Vec<Binary>,
+    message: Binary,
+    signature: Binary,
+) -> Result<bool, String> {
+    let aggregate_sig = match AggregateSignature::deserialize(signature.as_slice()) {
+        Ok(aggregate_sig) => aggregate_sig,
+        Err(e) => return Err(format!("{:?}", e)),
+    };
+    let pubkeys_result = public_keys
+        .iter()
+        .map(|pkb| match PublicKey::deserialize(pkb.as_slice()) {
+            Ok(pk) => Ok(pk),
+            Err(e) => return Err(format!("{:?}", e)),
+        })
```

Comment:
This match shouldn't be needed, since the error is later converted to a `String` anyways.

```suggestion
        .map(|pkb| PublicKey::deserialize(pkb.as_slice()))
```